net-p2p/bitcoin-core/bitcoin-core-25.1-r1.ebuild

```diff
@@ -144,23 +181,41 @@ src_configure() {
 		--enable-util-tx
 		--${wallet}-util-wallet
 		--disable-util-util
+		--without-seccomp
 		$(use_with libs)
 		$(use_with daemon)
-		$(use_with gui)
+		$(use_with gui gui qt5)
 		$(use_with dbus qtdbus)
 		$(use_with system-leveldb)
 		$(use_with system-libsecp256k1)
 	)
 	econf "${myeconfargs[@]}"
 }
 
+src_test() {
+	emake check
+
+	if use daemon ; then
+		local jobs=$(get_makeopts_jobs unset)
+		[[ "${jobs}" == unset ]] && unset jobs
```

Comment:
Is there a reason to not just make it 1 if unset?

Reply:
The upstream default is not 1.

Reply:
While upstream seems to use 4 as default, and unless I am missing something, I am not sure if this alone is enough to introduce this rather complex construct.

I think it's fine to simply unconditionally use the value returned by `get_makeopts_jobs`, or `get_makeopts_jobs 4` if you must.

Reply:
It's certainly less surprising for the build not to attempt to parallelize the tests if `MAKEOPTS` doesn't specify `-j`. Done.

Reply:
Exactly. Thanks!